frontend/src/components/CharacterArt.vue

```diff
@@ -231,6 +231,7 @@ export default {
       this.scene.add(light);
     },
     setupModel() {
+      if(this.scene === null || this.scene === undefined) return;
```

Comment:
is there a reason this can't be `if(!this.scene)`?

Reply:
Nope, not sure why I did this way.